x-pack/plugins/index_lifecycle_management/server/routes/api/policies/register_create_route.ts

```diff
@@ -32,7 +32,7 @@ async function createPolicy(
  * We only specify a rough structure based on https://www.elastic.co/guide/en/elasticsearch/reference/current/_actions.html.
  */
 const bodySchema = schema.object({
-  name: schema.string(),
+  name: schema.string({ maxLength: 1000 }),
```

Comment:
looks like we also have some validation on the client, but it's good to have a check here too 👍